docs/Makefile.am

```diff
@@ -33,21 +33,20 @@ endif
 
 # EXTRA_DIST breaks with $(abs_builddir) so build it using this variable
 # but distribute it (using the relative file name) in the next variable
-man_MANS = $(abs_builddir)/curl.1
-noinst_man_MANS = $(MK_CA_DOCS)
-dist_man_MANS = $(CURLCONF_DOCS) $(MK_CA_DOCS)
+man_MANS = $(abs_builddir)/curl.1 curl-config.1
+noinst_MANS = mk-ca-bundle.1
 CURLPAGES = curl-config.md mk-ca-bundle.md
 
 # Build targets in this file (.) before cmdline-opts to ensure that
 # the curl.1 rule below runs first
 SUBDIRS = . cmdline-opts libcurl
 DIST_SUBDIRS = $(SUBDIRS) examples
 
-CLEANFILES = $(man_MANS) curl.1 $(CURLCONF_DOCS) $(MK_CA_DOCS)
-nodist_MANS = $(CLEANFILES)
+CLEANFILES = curl.1
 
 EXTRA_DIST =                                    \
  $(CURLPAGES)                                   \
+ $(CURLCONF_DOCS)                               \
```

Comment:
Should mk-ca-bundle.1 be distributed as well, or no?

Reply:
Should it? It's not installed, and somewhere I think we should draw the line. You can easily build it....

Reply:
I wasn't sure, that's why I asked. It is in current master, so maybe the commit message should mention that it drops mk-ca-bundle.1 from dist.

Reply:
:+1: